components/TextField/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Platform, StyleSheet, Text, View, TextInput } from 'react-native';
+
+export default class TextField extends React.Component {
+  static propTypes = {
+    numberOfLines: PropTypes.number,
```

Comment:
let's have this take input.value and input.onChange from redux-form Field as props, and use them